app/components/questions/RideshareTripsOHare.js

```diff
@@ -0,0 +1,132 @@
+import { useState, useEffect } from 'react'
+import useFetch from 'use-http'
+import { Table } from '../../components/Table'
+import { Notification } from '../../components/Notification'
+import { FailureNotification } from '../../components/Notification'
+import {
+  Formatter,
+  calculatePercentChange,
+} from '../../site/metrics'
+
+const RIDESHARE_ENDPOINT = `${process.env.NEXT_PUBLIC_API}/question/ohare/rideshare`;
+  
+const MAX_PCT_CHANGE = -0.76;
+const RIDESHARE_DROPOFF_ALL = [
+  {
+    key: "dropoff_community_area",
+    name: "Area Number",
+  },
+  {
+    key: "area_name",
+    name: "Community Area",
+  },
+  {
+    key:"part",
+    name:"Part"
+  },
+  {
+    key: "total_trips",
+    name: "Dropoffs in 2019",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "total_trips_since",
+    name: "Dropoffs in 2020",
+    format: Formatter.numberWithCommas,
+    rowClasses: ["right"],
+  },
+  {
+    key: "pct_change",
+    name: "Change",
+    format: Formatter.percentChangeWithNoDecimal,
+    rowClasses: ["right"],
+    rgb: "209, 31, 0",
+    alpha: (v) => (v / MAX_PCT_CHANGE),
+  },
+];
+
+function addTotalTripsYear(before,since) {
+
+  var ds = {};
+  for(var key in before) {
+    ds[before[key]["dropoff_community_area"]] = before[key];
```

Comment:
If you want, you could do the same thing you do below and rename this "total_trips_before" instead of just "total_trips"